src/components/CartItem/CartItem.scss

```diff
@@ -0,0 +1,74 @@
+.cartItem {
+  width: 288px;
```

Comment:
Don't hardcode width, use 100% instead to fit whole window to make it responsive